test/cli.js

```diff
@@ -364,6 +364,14 @@ test('bails when config contains `"tap": true` and `"watch": true`', t => {
 	});
 });
 
+test('bails when config does not contain `--concurrency` and `-c` values', t => {
```

Comment:
@sindresorhus I read the test code in detail to understand how tests are written for `cli`! Do we need to write it in a similar way as we do for `watcher` et al? Any specific cases to cover?

cc/ @novemberborn